EmailNotifications/README.md

```diff
@@ -19,17 +19,25 @@ To send notifications, simply add the following code block to the behaviour wher
 
 ```C#
 
-    string to = "";
-    string subject = "";
-    string body = "";
+    //Set as true to send email with Omnia default smtp server. Set as false to send with a custom smtp server
+    bool sendByApi = true;
```

Comment:
I suggest inverting. Instead of using sendByApi, we can name it `usingExternalSmtpServer`. That argument should be optional, and if the user doesn't send it, we should consider the default value "False".